zetta_utils/mazepa_layer_processing/common/volumetric_callable_operation.py

```diff
@@ -51,30 +51,28 @@ def with_added_crop_pad(self, crop_pad: IntVec3D) -> VolumetricCallableOperation
         return attrs.evolve(self, crop_pad=self.crop_pad + crop_pad)
 
     def __attrs_post_init__(self):
-        output_crop_raw = self.crop_pad / self.res_change_mult
-        for e in output_crop_raw:
+        input_crop_pad_raw = self.crop_pad / self.res_change_mult
```

Comment:
Shouldn't this be `input_crop_pad_raw = self.crop_pad * self.res_change_mult`?

Reply:
You're correct, will fix.